src/site/content/en/blog/how-mercadolibre-optimized-web-vitals/index.md

```diff
@@ -0,0 +1,309 @@
+---
+layout: post
+title: How Mercado Libre optimized for Web Vitals (TBT/FID)
+subhead: Optimizing interactivity of Mercado Libre's product details pages.
```

Comment:
```suggestion
subhead: Optimizing interactivity of product details pages for a 90% reduction in First Input Delay and a 9% improvement in Chrome User Experience Report.
```
WDYT about mentioning the metric improvement here in the subtitle as well?

Reply:
It sounds like a good idea, but it should be: "for a 90% reduction in Reduction in Max Potential FID in Lighthouse and a 9% improvement in FID in Chrome User Experience Report."

I don't think I can apply a change to your suggestion from the UI and commit. Let me know if I should commit it on my end.

Reply:
Fixed!